test/suites/integration/bsp/submit-proofs.test.ts

```diff
@@ -168,28 +169,31 @@ describeBspNet(
       );
     });
 
-    it(
-      "BSP stops storing last file",
-      { skip: "Not implemented yet. Needs RPC method to build proofs." },
-      async () => {
-        // TODO: Build inclusion forest proof for file.
-        // TODO: BSP-Three sends transaction to stop storing the only file it has.
-        console.log(fileMetadata);
-        // // Build transaction for BSP-Three to stop storing the only file it has.
-        // const call = bspThreeApi.sealBlock(
-        //   bspThreeApi.tx.fileSystem.bspStopStoring(
-        //     fileMetadata.fileKey,
-        //     fileMetadata.bucketId,
-        //     fileMetadata.location,
-        //     fileMetadata.owner,
-        //     fileMetadata.fingerprint,
-        //     .fileSize,
-        //     false
-        //   ),
-        //   bspThreeKey
-        // );
-      }
-    );
+    it("BSP stops storing last file", async () => {
```

Comment:
I would rename this tests to something like "BSP requests to stop storing the last file", since we're not actually checking anything else